apps/web-mzima-client/src/app/settings/surveys/create-field-modal/create-field-modal.component.scss

```diff
@@ -7,14 +7,53 @@
   justify-content: flex-start;
 }
 
+// HTML Button
+.list-item1 {
+  width: 220px;
+  border: none;
+  margin: 0 8px 36px 8px;
+  font-size: 14px;
+  line-height: 1.3;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  color: var(--color-neutral-90);
+  background: var(--color-neutral-10);
+  box-shadow: inset 0 0 0 1px var(--color-neutral-10);
+
+  &--disabled {
+    cursor: default;
+    color: var(--color-neutral-60);
+
+    h3 {
+      color: inherit;
+    }
+  }
+}
+
+.list-item1 h3 {
+  margin: 0 0 0.5rem 0;
+  color: var(--color-black);
```

Comment:
I like how you created and used a separate `list-item1` class for the html button @kivuvarosekivuvan

Reply:
@Ifycode Thanks